lighthouse-core/test/report/html/renderer/report-renderer-test.js

```diff
@@ -135,8 +135,7 @@ describe('ReportRenderer', () => {
       const output = renderer.renderReport(sampleResults, container);
 
       const warningEls = output.querySelectorAll('.lh-run-warnings > ul > li');
-      assert.strictEqual(warningEls.length, 1);
-      assert.ok(/Links.*unsafe/.test(warningEls[0].textContent), 'did not add warning text');
+      assert.strictEqual(warningEls.length, sampleResults.runWarnings.length);
```

Comment:
there's already a "no `lighthouseRunWarnings`" test above this, so probably want to make a `const warningResults = Object.assign({}, sampleResults, {runWarnings: ['Warnings are great!']});` or whatever and pass that in